server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -166,6 +167,20 @@ export class AlfredResourcesFactory implements utils.IResourcesFactory<AlfredRes
 
         const tenantManager = new services.TenantManager(authEndpoint);
 
+        const throttleMaxRequestsPerMs = config.get("alfred:throttling:maxRequestsPerMs") as number || 1000000;
```

Comment:
Rather than using constants, we should have a "defaultThrottling" object where values are set to 1000000. And then we can import it everywhere

Reply:
Would it be better to make the params to Throttler and ThrottlerHelper optional with defaults instead?

Reply:
That also works if the values are only used inside that implementation.